docs/.vuepress/components/processing.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+    <div class="notification-container">
+        <div class="processing-tip">
+            <p>⚡当前语言可能更新不及时，最新信息请以<a href="/" target="_blank" rel="noopener noreferrer">英文文档</a>为准。如果翻译有问题，请<a href="https://github.com/flarum/docs/issues" target="_blank" rel="noopener noreferrer">告诉我们</a>。</p>
+        </div>
+    </div>
+</template>
```

Comment:
Might be good to ensure an ending newline for sake of convention, but that's super minor and nitpicky :grin:

Reply:
Nothing to it~ 😄